reference-data-proxy/src/v1/controllers/premium-schedule.controller.ts

```diff
@@ -29,11 +37,8 @@ const postPremiumSchedule = async (premiumSchedulePayload: any) => {
   try {
     const response = await axios({
       method: 'post',
-      url: `${mdmEAurl}/premium/schedule`,
-      auth: { username, password },
-      headers: {
-        'Content-Type': 'application/json',
-      },
+      url: `${mdm}/premium/schedule`,
```

Comment:
In a lot of other places we've used `process.env.APIM_MDM_URL` as if it is already going to have a slash at the end, but in this case and lower down in the file we've added a slash in the code. Is that deliberate?

Reply:
Good spot, have removed forward slash.